tests/php/PHPCodeSnifferBearTest.py

```diff
@@ -44,3 +62,19 @@
     valid_files=(good_file,),
     invalid_files=(bad_file,),
     tempfile_kwargs={'suffix': '.php'})
+
+
+class PHPCodeSnifferBearLineLengthTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section('name')
+        self.uut = PHPCodeSnifferBear(self.section, Queue())
+        test_files = os.path.join(os.path.dirname(
+            __file__), 'codesniffer_test_files')
+        self.config_file = os.path.join(test_files, 'config.xml')
+        self.long_length_file = os.path.join(
+            test_files, 'long_line_length.php')
+
+    def test_absolute_line_length(self):
+        self.section.append(Setting('max_line_length', '160'))
+        self.check_validity(self.uut, [], self.long_length_file)
```

Comment:
I think you can probably use `verify_local_bear` in this case, makes your test more compact :+1: